src/section.rs

```diff
@@ -0,0 +1,20 @@
+/// Positional arguments.
+#[derive(Debug, Clone)]
+pub struct Sec {
```

Comment:
I'm a bit on the fence about the naming. I know `Sec` to usually refer to "security", which might still make sense in a man page context. Perhaps calling this `Section` would remove any ambiguities.